mllib/src/main/scala/org/apache/spark/ml/classification/DecisionTreeClassifier.scala

```diff
@@ -85,18 +82,16 @@ final class DecisionTreeClassifier
   }
 
   /** (private[ml]) Create a Strategy instance to use with the old API. */
```

Comment:
Is it useful to mention `(private[ml])` in the JavaDoc? This seems to be duplicated info.